README.md

```diff
@@ -103,9 +103,6 @@ fortitude check --help
 
 ### Fixes
 
-> [!NOTE]
-> Added in v0.6.0
-
```

Comment:
Ah, I think maybe keep these to encourage people to upgrade!

Reply:
Good point, and the first time I've had a reason to use `git revert` in ages